iphone/cli/hooks/package.js

```diff
@@ -224,6 +224,15 @@ exports.init = function (logger, config, cli) {
 		exportsOptions.provisioningProfiles = {};
 		exportsOptions.provisioningProfiles[builder.tiapp.id] = pp.uuid;
 
+		// check if the app is using CloudKit
+		const entitlementsFile = path.join(builder.buildDir, builder.tiapp.name + '.entitlements');
+		if (fs.existsSync(entitlementsFile)) {
+			const plist = new appc.plist(entitlementsFile);
+			if (Object.keys(plist).some(function (key) { return key === 'com.apple.developer.icloud-container-identifiers'; })) {
```

Comment:
Linting error in Jenkins:
```
iphone/cli/hooks/package.js
-> 231:49  error  This line has 2 statements. Maximum allowed is 1  max-statements-per-line
```
This will be so awesome once we integrate Danger.js 🚀

Reply:
Fixed!